test/unit/base.test.js

```diff
@@ -129,6 +129,23 @@ describe('The converter must convert a swagger object', function() {
   });
 });
 
+describe('The converter must convert a swagger object which only have root path.', function() {
```

Comment:
The describe + it blocks' descriptions should read like a sentence

Check the last describe block in this file. We're already testing for two specific flows - this can be added as a third one there.